### PR TITLE
wike: init at 1.5.6

### PR DESCRIPTION
**Pull request by samalws**
###### Motivation for this change
Add https://github.com/hugolabe/Wike

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 137254` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>wike</li>
  </ul>
</details>

File: pkgs/applications/misc/wike/default.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    requests
+  ]);
+in stdenv.mkDerivation rec {
+  pname = "wike";
+  version = "1.5.6";
+
+  src = fetchFromGitHub {
+    owner = "hugolabe";
+    repo = "Wike";
+    rev = "dba6e1c01ae3f2b599d51db77a6cbe56e87dac3a";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/wike/default.nix:22]: Please use a release tag.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/wike/default.nix [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    ninja
+  ];
+
+  buildInputs = [
+    glib
+    appstream-glib
+    desktop-file-utils
```
Comment (jtojnar) [inline pkgs/applications/misc/wike/default.nix:35]: Should likely go to `nativeBuildInputs`. Same with `wrapGAppsHook` and `gobject-introspection`.

File: pkgs/applications/misc/wike/default.nix [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    requests
+  ]);
+in stdenv.mkDerivation rec {
+  pname = "wike";
+  version = "1.5.6";
+
+  src = fetchFromGitHub {
+    owner = "hugolabe";
+    repo = "Wike";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/wike/default.nix:22]:
```suggestion
    rev = version;
```

**Pull request merged**